cloudferrylib/os/actions/cinder_database_manipulation.py

```diff
@@ -17,8 +17,13 @@
 from cloudferrylib.base.action import action
 from cloudferrylib.utils import utils
 import jsondate
+from fabric.context_managers import settings
+from cloudferrylib.utils import remote_runner
 
 NAMESPACE_CINDER_CONST = "cinder_database"
+COPYING_ATTEMPTS = 5
+GET_CINDER_FOLDER_CMD = ("mount | grep `cat /etc/cinder/nfs* | head -n 1` | "
```

Comment:
Hardcoded values for NFS config should be replace with more sophisticated logic to handle different configurations. I'm 100% sure this will break the very first time we run it in any prod or staging environment. Please refactor to pick correct values from config at least.